src/calendardata.h

```diff
@@ -54,6 +55,8 @@ struct Notebook {
     QString uid;
     QString description;
     QString color;
+    int accountId;
```

Comment:
I would initialize accountId explicitly to a value that indicates an invalid/non-existent account. Relying on zero-initialization makes me feel a bit paranoid :)

Alternatively, the accountId could be primed in NemoCalendarWorker::loadNotebooks() in cases where there is no valid account assigned to a notebook.